pytest_fixtures/broker.py

```diff
@@ -39,3 +41,17 @@ def rhel77_contenthost_class(request):
     with VMBroker(host_classes={'host': ContentHost}, **BROKER_RHEL77) as host:
         request.cls.content_host = host
         yield
+
+
+@pytest.fixture
+def satellite_latest():
+    """A fixture that provides a latest Satellite"""
+    with VMBroker(host_classes={'host': Satellite}, workflow='deploy-sat-lite') as sat:
+        yield sat
+
+
+@pytest.fixture
+def capsule_latest():
+    """A fixture that provides an unconfigured latest Capsule"""
+    with VMBroker(host_classes={'host': Capsule}, workflow='deploy-sat-lite') as sat:
```

Comment:
I think you mean `deploy-sat-capsule` here for the workflow.

Reply:
ha yes!!